New Todo UI for FlightJS

kmargaritis opened the pull request:
Upgraded FlightJS to new Todo ui. 

* jasmine-flight and flightjs are linked to git repos

sindresorhus commented on examples/flight/package.json:
```diff
@@ -1,17 +1,30 @@
-  }
+	"name": "flight-todomvc",
+	"private": true,
+	"version": "0.0.1",
```
`name` and `version` isn't needed

sindresorhus commented on examples/flight/package.json:
```diff
@@ -1,17 +1,30 @@
+	"private": true,
+	"version": "0.0.1",
+	"dependencies": {
+		"depot": "~0.1.6",
```
use `^` on all of these (instead of `~`).

sindresorhus commented on examples/flight/bower.json:
```diff
@@ -2,15 +2,9 @@
-    "jasmine-jquery": "~2.0.5"
+    "jasmine-flight": "~4.0.0"
   }
 }
```
bower.json can be removed now

sindresorhus commented on examples/flight/.gitignore:
```diff
@@ -1 +1,40 @@
+!node_modules/requirejs-text/text.js
+
+node_modules/todomvc-app-css/*
+!node_modules/todomvc-app-css/index.css
```
You need to actually include these files.

kmargaritis commented:
This is exactly what happens when you are sending PRs seconds before leaving for a coffee!

arthurvr commented on examples/flight/.gitignore:
```diff
@@ -1 +1,40 @@
-node_modules/
+node_modules/.bin/*
+
+node_modules/depot/*
```
The start at the end isn't really needed. Ignoring the folder itself will work too.

kmargaritis commented:
@sindresorhus I think it is ok. @arthurvr i didn't have time to remove the stars. Thanks for the heads up!

kmargaritis commented:
@passy How can you check from Github which commits were not merged?

kmargaritis commented:
@sindresorhus I saw you reverted this. I have updated the transient deps, put back the asterisks, and squashed the whole thing. Do you want to sent a new PR?

sindresorhus commented:
Hmm, where did I revert this?

kmargaritis commented:
[Here](https://github.com/tastejs/todomvc/commit/1e2fcbf1ba375fca43c398943e360a0e7f1db6ec)

sindresorhus commented:
Oh, gawd... I didn't mean to push that commit. Fixed now.

sindresorhus commented:
@kmargaritis Any idea why https://github.com/tastejs/todomvc/commit/c898a78d00ab80b4b2f91c45fa9fe4baebd3a482 contains so many changes?

kmargaritis commented:
No idead @sindresorhus. Probably @passy has a clue. This is what i send [b6c18e2](https://github.com/tastejs/todomvc/commit/b6c18e2e8b602d2870f3018989ba250d2b475972)

passy commented:
The commits are all in, but I squashed them before merging. I still have no idea what happened with that mega commit. Went through some theories with Sindre, but I'm still in the dark. Sorry for the poopoo.
